Makefile

```diff
@@ -612,19 +612,15 @@ isti%.yaml: $(HELM)
 		  --values install/kubernetes/helm/istio/values-$@ \
 		  install/kubernetes/helm/istio > install/kubernetes/$@
 
-deploy/all: $(HELM)
-	kubectl create ns istio-system > /dev/null || true
+sidecar-injector.yaml: $(HELM)
```

Comment:
I don't think we need a separate yaml file - users should specify if they want or not sidecar in values.yaml or --set